### PR TITLE
Release notes for 19.9.0

### PR DESCRIPTION
**Pull request by viraj2712**

### CODE REVIEW

File: source/partials/release_notes/_release-19-9-0.md.erb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<h4>Java 8 and 10 support removed</h4>
+
+Support for Java 8 and 10 has been dropped from this release. This is in line with the Java Licensing Changes we had announced earlier this year in this [blog post](https://www.gocd.org/2019/05/21/official-stance-on-java/).
+
+Starting 19.9.0, GoCD Server and Agent would run either on Java 11 or 12. Support for Java 13 will be introduced soon. GoCD installers from 19.6.0 come with Java Runtime pre-bundled, so users no longer need to install it separately.
```
Comment (adityasood) [inline source/partials/release_notes/_release-19-9-0.md.erb]: > GoCD Server and Agent would run either on Java 11 or 12

GoCD Server and Agent would only support Java 11 or 12

File: source/partials/release_notes/_release-19-9-0.md.erb [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<h4>Java 8 and 10 support removed</h4>
+
+Support for Java 8 and 10 has been dropped from this release. This is in line with the Java Licensing Changes we had announced earlier this year in this [blog post](https://www.gocd.org/2019/05/21/official-stance-on-java/).
```
Comment (adityasood) [inline source/partials/release_notes/_release-19-9-0.md.erb]: This is in line with our [official stance on Java](https://www.gocd.org/2019/05/21/official-stance-on-java/) we announced earlier this year post the Java Licensing Changes.

File: source/partials/release_notes/_release-19-9-0.md.erb [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<h4>Java 8 and 10 support removed</h4>
```
Comment (adityasood) [inline source/partials/release_notes/_release-19-9-0.md.erb]: Support for Java 8 and 10 removed

File: source/partials/release_notes/_release-19-9-0.md.erb (hunk 1 above)
Comment (varshavaradarajan) [reply source/partials/release_notes/_release-19-9-0.md.erb]: Starting 19.9.0, GoCD Server and Agent ~~would~~ run/support ...

Comment (viraj2712) [reply source/partials/release_notes/_release-19-9-0.md.erb]: done

File: source/partials/release_notes/_release-19-9-0.md.erb (hunk 2 above)
Comment (viraj2712) [reply source/partials/release_notes/_release-19-9-0.md.erb]: done

File: source/partials/release_notes/_release-19-9-0.md.erb (hunk 3 above)
Comment (viraj2712) [reply source/partials/release_notes/_release-19-9-0.md.erb]: done

**Pull request merged**